src/components/search/SearchResultsRow.tsx

```diff
@@ -16,10 +16,31 @@ const createScroller = ({ title = '' }) => ({
 </div>`
 });
 
+type CardOptions = {
+    itemsContainer?: HTMLElement,
+    parentContainer?: HTMLElement,
```

Comment:
Isn't `HTMLDivElement` better (stricter)?
So we won't be able to pass buttons/images/etc.

Reply:
In my opinion it's best to keep things as generic as possible until there is a reason to be more strict... :man_shrugging: